relayer/go.mod

```diff
@@ -5,7 +5,8 @@ go 1.18
 require (
 	github.com/NethermindEth/juno v0.0.0-20220630151419-cbd368b222ac
 	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1
-	github.com/dontpanicdao/caigo v0.2.1-0.20220517132056-e34006317632
+	github.com/dontpanicdao/caigo v0.3.0
+	github.com/ethereum/go-ethereum v1.10.18
```

Comment:
Core just bumped to 1.10.20 - or better yet, could this import be avoided?

Reply:
removed! but there's still an indirect import because `caigo` imports it

Reply:
:homer-shakes-fist: